iceoryx_hoofs/test/integrationtests/test_resizeable_lockfree_queue_stresstest.cpp

```diff
@@ -336,6 +339,8 @@ class ResizeableLockFreeQueueStressTest : public ::testing::Test
     Queue sut;
 
     std::chrono::seconds runtime{3};
+    const iox::units::Duration m_fatalTimeout = 60_s;
+    Watchdog m_watchdog{m_fatalTimeout};
```

Comment:
Something like `const iox::units::Duration m_fatalTimeout = 60_s + ox::units::Duration(runtime);` might save our future self from swearing

Reply:
In principle a good idea, but this does not work with the given conversions. I could introduce another var but would prefer not to.